pkgs/data/themes/adi1090x-plymouth-themes/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nix-prefetch jq
+
+theme_archives=$(curl --silent https://api.github.com/repos/adi1090x/files/git/trees/master\?recursive=1 \
```

Comment:
This needs to optionally take a GITHUB_TOKEN because we easily run into api rate limits

Reply:
Added this to update.sh and tested with a fine-grained access token with only the Public Repositories (read only) permission.